docs/src/main/asciidoc/azure-functions-resteasy-guide.adoc

```diff
@@ -0,0 +1,113 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+
+include::./attributes.adoc[]
+= {project-name} - Using Resteasy with Azure Functions (Serverless)
+
+Writing in Java standards rather than cloud provider specific frameworks like Azure Functions for Java keeps your REST
+microservices portable to other cloud providers and other environments.
+
+This guide demonstrates how you can deploy a {project-name} JAX-RS microservice as
+an Azure Function.  You can develop your REST endpoints using JAX-RS and Resteasy, but still
+be able to leverage the cheaper price points, scalability, and ease of deployment of Azure Functions.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* JDK 1.8 (Azure requires JDK 1.8)
+* Apache Maven 3.5.3+
+* A completed https://quarkus.io/getting-started[Getting Started Guide] for Quarkus.
+* An Azure Account
+* Azure CLI Installed
+
+== Solution
+
+This guide requires that you have finished creating the JAX-RS application in the Getting Started Guide.
+You will create a simple maven project that references the jar artifact you created for your JAX-RS application.
+
+== Creating the Maven Project
+
+First create a new directory called `azure` and change your terminal's working directory to it.
+
+[source, subs=attributes+]
+----
+$ mkdir azure
+$ cd azure
+----
+
+
+Next create the azure maven project for your Quarkus application.
+
+[source, subs=attributes+]
+----
+mvn archetype:generate \
+    -DarchetypeGroupId=io.quarkus \
+    -DarchetypeArtifactId=quarkus-azure-functions-resteasy-archetype \
+    -DarchetypeVersion={quarkus-version}
+----
+
+Running this command will run maven in interactive mode and it will ask you to fill in some build properties:
+* groupId - The groupId of your existing quarkus application.  Since we are deploying the Getting Started Guide to
+  azure, this will be `org.acme`.
+* artifactId - The artifactId of your existing quarkus application.  Since we are deploying the Getting Started Guide
+  this will be `getting-started`
+* version - Version that should match your existing quarkus application artifact.
+* package - use the default value
+* azureFunction - Name of your azure function. Can be anything you want.  Type in `getting-started`.
+* azureFunctionAppName - Application name of your azure function.  Must be unique subdomain name under *.azurewebsites.net. Otherwise deploying to Azure will fail.
+* azureFunctionAppRegion - Dependent on your azure region.  You can try `westus` if you don't know this value.
+* azureFunctionResourceGroup - Any value you want.
+
+This will generate a pom.xml that generates an artifactId with a suffix of `-azure`.  You can modify any of the values specified
+above in the pom.xml file, but you will need to completely rerun the build.
```

Comment:
Shouldn't it be `pom.xml` (with backticks)? I think, the same could be said of every item name in the previous list, no?